spec/lib/vcr/errors_spec.rb

```diff
@@ -110,7 +126,7 @@ def request_with(values)
           end
         end
 
-        it 'does not mention :allow_playback_repeats if the current cassette does not have a used matching interaction' do
+        it 'does not mention :allow_playback_repeats if the cassette does not have a used matching interaction' do
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.